Casks/c/cables-gl.rb

```diff
@@ -0,0 +1,27 @@
+cask "cables-gl" do
```

Comment:
```suggestion
cask "cables" do
```

This change also requires renaming the file itself.